ivy/array/elementwise.py

```diff
@@ -1,275 +0,0 @@
-# global
```

Comment:
I'm not sure deleting the file from the PR removes it from the PR. It is still listed as an altered file.